%YAML 1.2
---

corner_num_x : 7
corner_num_y : 6
square_size  : 0.0286